Label-Microservice/requirements.txt

```diff
@@ -33,15 +33,19 @@ fastai==1.0.55
 fastprogress==0.1.21
 fire==0.1.3
 future==0.17.1
+github3.py==1.3.0
```

Comment:
See the comment I made about Jeremy's modules at the root of /py

Having a seperate requirements.txt here and there at the same time is confusing as it is not clear which dependencies control especially if this application will rely on those general utilities.